Makefile

```diff
@@ -12,6 +12,10 @@ BIN = node_modules/.bin
 s:
 	$(BIN)/coffee index.coffee
 
+# Start the server using forever
+sf:
+	$(BIN)/forever $(BIN)/coffee --nodejs --max_old_space_size=960 index.coffee
```

Comment:
You can take the `--max_old_space_size=960` flag off here. That's an artifact of debugging the memory leak on Force. Luckily Positron averages less than a quarter of this memory use anyways so it wouldn't affect anything.